debian/bullseye/foreman/control

```diff
@@ -14,7 +14,7 @@ Package: foreman
 Architecture: any
 Section: web
 Priority: extra
-Depends: ruby, ruby (>= 1:2.7) | bundler (>= 1.3), ruby-dev, zlib1g-dev,
+Depends: ruby (>= 1:2.7), ruby-dev, zlib1g-dev,
```

Comment:
ruby only provides `/usr/bin/bundle$(RUBY_VERSION)`. Looking at our packaging I think we consistently use that, but I wonder if anything does use `bundle` at runtime.

Reply:
Correct, but in the current notation, `bundler` is not installed anyways on `ruby >= 2.7` (unless pulled in otherwise).

If things break at runtime, we should have test for that. So :woman_shrugging:

Reply:
But your comment is somewhat right, as we do *not reliably* use `bundle$(RUBY_VERSION`… adding a change for that!